.github/workflows/build-docker-test.yml

```diff
@@ -0,0 +1,63 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
This file has been already created in https://github.com/0xPolygonHermez/zkevm-contracts/blob/feature/ongoingPP/.github/workflows/build-docker.yml where all the unnecessary data from gh secrets has been removed.
I would import that exact file, so it will be easier if we do a rebase afterwards